docs/topics/rn-known-issues.adoc

```diff
@@ -16,48 +16,23 @@ For a complete list of all known issues, see the list of link:https://issues.red
 |Component
 |Summary
 
-|link:https://issues.redhat.com/browse/WINDUPRULE-598[WINDUPRULE-598]
-|VS Code Extension
-|Quick Fix does not fix XML rules.
-
-|link:https://issues.redhat.com/browse/WINDUPRULE-589[WINDUPRULE-589]
-|VS Code Extension
-|Right-clicking a VS Code configuration and selecting *Open* does not always open the configuration.
-
-|link:https://issues.redhat.com/browse/WINDUPRULE-516[WINDUPRULE-516]
-|VS Code Extension
-|Quick Fix does not fix multiple rules.
-
-|link:https://issues.redhat.com/browse/WINDUP-2709[WINDUP-2709]
-|Eclipse Plugin
-|Installing the Eclipse plugin 5.0.0 on CodeReady Studio 12.16 requires Maven.
-
-|link:https://issues.redhat.com/browse/WINDUP-2692[WINDUP-2692]
-|Web UI & Windup-as-a-Service
-|When RHAMT 4.3.1 is deployed on OpenShift Container Platform with the `deploy.sh` script, an invalid route is created in the namespace.
+|link:https://issues.redhat.com/browse/WINDUP-3138[WINDUP-3138]
+|Eclipse Che plugin
+|MTA plugin does not work with CodeReady Studio version 2.10.
 
-|link:https://issues.redhat.com/browse/WINDUP-2645[WINDUP-2645]
-|Eclipse Che Plugin
-|Quick Fix does not fix multiple rules.
+|link:https://issues.redhat.com/browse/WINDUP-3019[WINDUP-3019]
+|Migration-Application
+|The bash completion feature does not work.
 
-|link:https://issues.redhat.com/browse/WINDUP-2632[WINDUP-2632]
-|VS Code Extension
-|`--sourceMode` must be enabled by default.
+|link:https://issues.redhat.com/browse/WINDUP-3104[WINDUP-3104]
+|Web UI & Windup as a Service (windup-web)
+|Web console on MacOS;  After analysis, clicking a *Short reports* link results in a "Not found" page.
```

Comment:
Change component to "Web console installed on MacOS"

I do not see a "Short reports" link mentioned in the bug.
This looks like a grammatical dangler. Please rewrite it.

Reply:
The Jira's title is " Web Console on MacOS - After analysis clicking on the "short reports" link results in a "not found" page " 

Would "After you run an analysis, if you click a *Short reports* link, a "Not found" page displays." be better?

Reply:
IBM (usage section) says that "displays" is only used as a transitive verb, not intransitive. But in other contexts, IBM seems to allow "is displayed", so I think you can go with that. So change the last clause to "a `Not found` page is displayed".